### ISSUE TITLE
Getting response headers

### ISSUE THREAD

**Issue by kurko** [Jakenberg/binance-ruby#24]
With the current approach, all requests return only the response body, that is, a JSON that is converted into a hash. That means it's not possible to know information about rate limiting, cache and more.

Here's an example: https://github.com/Jakenberg/binance-ruby/blob/master/lib/binance/api/account.rb#L26-L28

Would there be interest to instead return the HTTP object and let the client call `response.body`, instead of prohibiting the client from accessing anything other than the HTTP body?

**Comment by Jakenberg** [comment]
Yeah that makes sense to me. If you'd like to put up a backwards compatible PR I'll review within a week or so.

**Comment by 0xjmp** [comment]
closed as stale.
